src/Symfony/Bundle/FrameworkBundle/Secrets/SodiumVault.php

```diff
@@ -82,12 +84,12 @@ public function seal(string $name, string $value): void
         $this->lastMessage = null;
         $this->validateName($name);
         $this->loadKeys();
-        $this->export($name.'.'.substr_replace(md5($name), '.sodium', -26), sodium_crypto_box_seal($value, $this->encryptionKey ?? sodium_crypto_box_publickey($this->decryptionKey)));
+        $this->export($name.'.'.substr(md5($name), 0, 6), sodium_crypto_box_seal($value, $this->encryptionKey ?? sodium_crypto_box_publickey($this->decryptionKey)));
 
         $list = $this->list();
```

Comment:
shouldn't we flock the file ?

Reply:
https://github.com/nicolas-grekas/symfony/blob/ba2148fff318fda5d4b0150e68c2264723415f9c/src/Symfony/Bundle/FrameworkBundle/Secrets/SodiumVault.php#L206

Reply:
I don't think we should - secrets are not updated live ayway.